### PR TITLE
BU-11: Add MusicBrainz database module to brainzutils

### PR DESCRIPTION
**Pull request by kartikeyaSh**
# Summary

* This is a…
    * [ ] Bug fix
    * [x] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem
MusicBrainz DB is being used by more than one MB projects (CritiqueBrainz, AcousticBrainz) for useful information and MessyBrainz also needs this.

* JIRA ticket (_optional_): [BU-11](https://tickets.metabrainz.org/browse/BU-11)

# Solution
 The code for accessing different entities can be shared by using brainzutils musicbrainz_db module.

# Action

Created musicbrainz_db module.

### CODE REVIEW

**Comment by kartikeyaSh** [comment]
I've written this with **python3** syntax. Does it have to work with both versions of python?

**Comment by paramsingh** [comment]
@kartikeyaSh, brainzutils is used by AB which is currently python2, so we don't want to break python2 compatibility.

**Comment by paramsingh** [comment]
@brainzbot add to whitelist.

**Comment by paramsingh** [comment]
Another option we have is documenting and testing the other serialize functions, but I think that this should be done in a different ticket anyways.

**Comment by paramsingh** [comment]
@rsh7 has addressed some comments in https://github.com/metabrainz/brainzutils-python/pull/16/commits/a203ed1ea8cec62d414773e19ce2dcb5c5506b9d if you wanna do a `git cherry-pick` or copypasta.

File: requirements.txt [hunk 1]
```diff
@@ -6,3 +6,5 @@ certifi
 redis==2.10.5
 msgpack-python==0.4.8
 requests==2.18.1
+SQLAlchemy==1.0.8
```
[the remark this answers is not in the document]
Comment (kartikeyaSh) [reply requirements.txt]: I used it just cz its the version being used by messybrainz. I don't have CB build, so I can't test it. Can you post some logs in which I can try to find out something? And we should try to use one version of libraries in all MB projects whenever possible. Is downgrading or upgrading the libraries a good thing in this case? In [CB requirements](https://github.com/metabrainz/critiquebrainz/commit/cd369f08d81191350789a2e111ce075683a49bb5#diff-b4ef698db8ca845e5845c4618278f29a) there is nothing on why it was upgraded in the first place. To me, it looks like its just a case of "trying to use new version just cz it has bug fixes etc."

File: requirements.txt [hunk 2]
```diff
@@ -6,3 +6,5 @@ certifi
...
+mbdata==2017.6.2
```
[the remark this answers is not in the document]
Comment (kartikeyaSh) [reply requirements.txt]: Once we settle on the version of SQLAlchemy I'll add that to `requirements_dev.txt`.

File: brainzutils/musicbrainz_db/__init__.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+from contextlib import contextmanager
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker, scoped_session, Session
+from sqlalchemy.pool import NullPool
+
+
+engine = None
+DEFAULT_CACHE_EXPIRATION = 12 * 60 * 60  # seconds (12 hours)
```
Comment (paramsingh) [inline brainzutils/musicbrainz_db/__init__.py]: this shouldn't be needed.

**Pull request merged**